src/IRsend.h

```diff
@@ -154,8 +154,9 @@ enum panasonic_ac_remote_model_t {
 
 /// Sharp A/C model numbers
 enum sharp_ac_remote_model_t {
-  A907 = 1,  // 802 too.
+  A907 = 1,
   A705 = 2,
+  A903 = 3,  // 820 too
```

Comment:
802 vs 820 is not a typo?

Reply:
The old "802" was the typo that needed fixing. Thanks for checking!